protected/config/.env.example

```diff
@@ -0,0 +1,3 @@
+DEBUG=1
```

Comment:
I'm a little worried that the ‘.env’ file can simply be called up via the browser if the web server is configured incorrectly. Are there common approaches against this? e.g. an ‘env.php’ as you have already mentioned?

Reply:
@luke- I placed it in protected directory, it's protected via htaccess, no one can open it via browser

Reply:
Ok, in our standard configurations, this directory should be protected. However, I have also seen some installations where this is not the case.  

Anyway, this is more of a power user feature and even optional. Perhaps we can also add a clear header to the `.env.example`.

e.g. 

```env
# Warning: This file may contain sensitive information. Make sure that this file is not
# distributed via the web server and does not have file permissions that are too open.
```
---

I am currently thinking about whether we should put the `.env[.example]` in the project root directory. Here it is more visible and we could also adjust the documentation/example configurations so that this file is also protected.

What do you think?

Reply:
@luke- good idea to put it to root, it will be visible to everyone. let's add this header. But anyway we can prevent opening this files via .htaccess too in same way that it's done for example for composer.json